kubedb/Dockerfile

```diff
@@ -7,4 +7,7 @@ COPY ./db_setup.sql .
 COPY ./db_setup.sh .
 
 RUN chmod +x db_setup.sh
+
+USER mssql
```

Comment:
that simple? if so, you're some kind of wizard!

Reply:
haha it was just a test, but yeah saw the https://github.com/Microsoft/mssql-docker/issues/121 and think that's better